src/LandIce/problems/LandIce_StokesFOThermoCoupled.hpp

```diff
@@ -430,7 +412,7 @@ constructEnthalpyEvaluators (PHX::FieldManager<PHAL::AlbanyTraits>& fm0,
   //Output
   p->set<std::string>("Residual Variable Name", resid_names[2]);
 
-  ev = createEvaluatorWithTwoScalarTypes<EnthalpyResid,EvalT>(p,dl,FST::Scalar,get_scalar_type(melting_temperature_name));
+  ev = createEvaluatorWithOneScalarType<EnthalpyResid,EvalT>(p,dl,FST::Scalar);
```

Comment:
Here too: if the scalar type is known at compile time, you save compilation time by not using the `createEvaluatorWithOneScalarType` helper function.

Reply:
OK